nautobot/dcim/tables/devices.py

```diff
@@ -605,6 +605,16 @@ class Meta(DeviceComponentTable.Meta):
 
 
 class DeviceInterfaceTable(InterfaceTable):
+    def __init__(self, is_vc_master=False, *args, **kwargs):
+        if is_vc_master:
+            self.Meta.default_columns.append("device")
```

Comment:
`Meta` is a class attribute, not an instance attribute, so I'm worried that this will bleed over to other instances. I think instead you should be able to do `self.columns.show("device")`? But see also my comment below about maybe doing this directly from the view instead of here.